### ISSUE TITLE
Import stuck at "Preparing for import"

### ISSUE THREAD

**Issue by ThRvnchst** [gnosygnu/xowa#199]
I tried to do an import of starwars.wikia.com using the Import/Offline page with an existing .bz2 dump file and the "Other Wiki" preset, but it always stays at "Preparing for import". This is the case with every wikia wiki I tried to import, unless I select the MediaWiki preset. Then I can fully import the dump and use the wiki for some reason. I redownloaded XOWA several times, reinstalled Java twice and did about everything else I can think of.

Here's my log files since my last reinstall.
[log.zip](https://github.com/gnosygnu/xowa/files/1617381/log.zip)

**Comment by gnosygnu** [comment]
Hi! Thanks for the report as well as the logs!

It looks like I broke the wikia import a few months ago. I made a quick fix for it, and tested it with the starwars.wikia.com. If you want to give it a try, the Windows 64 version is here: https://drive.google.com/open?id=1CxodcsqNcrnYawydePUMGkhWonAHwBBs

I'll put in a more permanent fix later, and incorporate that into the next release.

Hope this helps! Thanks!
